libseshat/tests/simple.c

```diff
@@ -91,9 +91,9 @@ void test_all( void )
             char test_service[32];
 
             sprintf(test_buffer, "WebPa1%d", cnt);
-            response = seshat_discover(test_buffer);
-            CU_ASSERT(NULL == response);
-            free(response);
+            url = seshat_discover(test_buffer);
+            CU_ASSERT(NULL == url);
```

Comment:
This is a different test, _test_none_. So, this is expected behavior. _test_all_ is still present.